### ISSUE TITLE
Photo facet filter based on route or area gear style

### ISSUE THREAD

**Issue by brendanheywood** [theCrag/website#3031]
**What happened?**

Be able to filter to photo based on what nodes they are linked to. We will more often have a vague link to a cliff rather than a route, so it will be a soft search probably related to the gear style stats for that node. ie we could say 'find photos of boulders in araps' and it should add a soft weighting or boost factor.

**What you expected:**

* [ ] fuzzy filter based on gear style

**Comment by scd** [comment]
@brendanheywood I want to tease out what you mean by fuzzy filter in this instance. If I understand you correctly this is a sort order thing.

**Comment by brendanheywood** [comment]
Yeah so with most search engine frameworks like solr or elastic search you can hard filter things out but for cases like this we just want to refine the sort order which is done by a 'boost' factor. The problem here is that if you filter by say sports routes we need to do that filtering against the node not the photo and then join, but a crag could be 80% sport and 20% trad, and because the photo might be linked to the area and not the exact route we can't get this exact.

So I instead I think we need something vaguely like if a an area is 95% sport then then boost the photos in it by some amount say 95x, but if an area is only 10% sport then boost it by say 10%. OR we could just be simple and brutal and so just filter to areas which contain some sport and do no boosting at all. 

This filter isn't urgent at all, I think there is much higher stuff like the forums work.
